pom.xml

```diff
@@ -25,6 +25,7 @@
 		<oracle.driver.jdkversion>8</oracle.driver.jdkversion>
 		<mssql.driver.version>7.4.1.jre8</mssql.driver.version>
 		<mysql.driver.version>8.1.0</mysql.driver.version>
+		<!-- Should upgrade to new MariaDB driver 3.1.4, 3.2.0 or newer for JDK17 support but currently not working with XA transaction managers in the framework. -->
```

Comment:
Dit is voor de 7.8 niet relevant toch?

Reply:
Je hebt gelijk, niet in deze bewoording. Alleen relevant voor 7.8 om niet te upgraden naar 3.1.x.

Ik zal het aanpassen.